frappe/website/doctype/website_theme/website_theme.py

```diff
@@ -119,14 +126,58 @@ def add_website_theme(context):
 		context.theme = website_theme or frappe._dict()
 
 def get_active_theme():
-	website_theme = frappe.db.get_value("Website Settings", "Website Settings", "website_theme")
+	website_theme = frappe.db.get_single_value("Website Settings", "website_theme")
 	if website_theme:
 		try:
 			return frappe.get_doc("Website Theme", website_theme)
 		except frappe.DoesNotExistError:
 			pass
 
 
-def get_scss(doc):
-	return frappe.render_template('frappe/website/doctype/website_theme/website_theme_template.scss', doc.as_dict())
 
+def get_scss(website_theme):
+	"""
+	Render `website_theme_template.scss` with the values defined in Website Theme.
+
+	params:
+	website_theme - instance of a Website Theme
+	"""
+	apps_to_ignore = tuple(d.app for d in website_theme.ignored_apps)
+	available_imports = get_scss_paths()
+	imports_to_include = [d for d in available_imports if not d.startswith(apps_to_ignore)]
```

Comment:
We cannot use `startswith` here. If I disable 'frappe' then 'frappe_docs' or 'frappe_io' would also be excluded.